apps/admin/src/app/sections/user/add-edit-user-form.tsx

```diff
@@ -21,6 +21,7 @@ const defaultValues = {
     lastName: '',
     email: '',
     phoneNumber: '',
+    status: '',
```

Comment:
I think we can make default status inactive

Reply:
still pending

Reply:
update status default inactive